nucliadb_utils/nucliadb_utils/const.py

```diff
@@ -51,6 +51,24 @@ class INDEX:
         subject = "node.{node}"
         group = "node-{node}"
 
+    class EXPORTS:
+        """
+        Exporting kbs
+        """
+
+        name = "nucliadb"
+        subject = "ndb.consumer.exports.{partition}"
+        group = "nucliadb-exports-{partition}"
+
+    class IMPORTS:
+        """
+        Importing kbs
+        """
+
+        name = "nucliadb"
+        subject = "ndb.consumer.imports.{partition}"
+        group = "nucliadb-imports-{partition}"
```

Comment:
is it OK to reuse the same "nucliadb" stream, but with different subjects?

Reply:
No, let's create a new stream please!

I think we are already abusing streams a bit.